test/views/TaxYearsViewSpec.scala

```diff
@@ -44,51 +44,51 @@ class TaxYearsViewSpec extends ViewSpec {
   "individual" when {
     "The TaxYears view with itsaSubmissionFeatureSwitch FS disabled" when {
       "the view is displayed" should {
-        s"have the title '${taxYears.title}'" in new Setup(List(testYearPlusOne, testYear)) {
+        s"have the title '${taxYears.title}'" in new Setup(List(testYearPlusOne, testTaxYear)) {
           document.title() shouldBe taxYears.title
         }
 
-        "have a header" in new Setup(List(testYearPlusOne, testYear)) {
+        "have a header" in new Setup(List(testYearPlusOne, testTaxYear)) {
           layoutContent.selectHead("h1").text shouldBe taxYears.heading
         }
       }
 
       "the user has two tax years" should {
-        "display two tax years" in new Setup(List(testYearPlusOne, testYear)) {
-          document.selectHead("dl div:nth-child(1) dt").text() shouldBe taxYears.taxYear(testYear.toString, testYearPlusOne.toString)
-          document.selectHead("dl div:nth-child(2) dt").text() shouldBe taxYears.taxYear((testYear - 1).toString, testYear.toString)
+        "display two tax years" in new Setup(List(testYearPlusOne, testTaxYear)) {
+          document.selectHead("dl div:nth-child(1) dt").text() shouldBe taxYears.taxYear(testTaxYear.toString, testYearPlusOne.toString)
```

Comment:
these selector strings are repeated everywhere and should be constants really to keep it DRY - admittedly not what you changed in the PR

Reply:
Agreed.